apps/SageAccountWeb/package.json

```diff
@@ -103,14 +103,6 @@
       "last 1 safari version"
     ]
   },
-  "resolutions": {
-    "js-yaml": "3.13.1",
-    "react": "18.2.0",
-    "react-hot-toast": "2.2.0",
-    "@types/react": "18.0.17",
-    "@types/react-dom": "18.0.6",
-    "minimatch": "^3.1.2"
-  },
```

Comment:
According to the output of `pnpm install`, these resolution entries are ignored. Only the base package.json `resolutions` are used.

Reply:
See docs for `overrides`: https://pnpm.io/package_json#pnpmoverrides  - this is the same feature, just a different property name.